moztelemetry/dataset.py

```diff
@@ -70,6 +71,9 @@ class Dataset:
         schema = ['submissionDate', 'docType', 'platform']
 
         records = Dataset(bucket, schema) \\
+            .select('clientId',
+                    os_name='environment.system.os.name',
```

Comment:
I added this line to the example `'payload.threadHangStats[].hangs[].stack[0:2]` which shows how to select a slice of an array nested under 2 other arrays. I don't think you can do that with `get_pings_properties` 😸